transformers/tyntecWhatsAppTransformer/transformer-v4.ts

```diff
@@ -148,7 +148,7 @@ const convertWebchatContentToWhatsApp = (processedOutput, sessionId: string, ses
 		for (let stackItem of processedOutput.outputStack) {
 
 			// check if default text was sent
-			if (stackItem.text && !(stackItem.data && stackItem.data?._cognigy && (stackItem.data?._cognigy?._default))) {
+			if (stackItem.text && !(stackItem && !stackItem.data?._cognigy?._default)) {
```

Comment:
simply (stackItem.text && !stackItem.data?._cognigy?._default) should do

Reply:
done